extensions/query-store/src/extension.ts

```diff
@@ -4,17 +4,20 @@
  *--------------------------------------------------------------------------------------------*/
 
 import * as vscode from 'vscode';
+import * as azdata from 'azdata';
 import { QueryStoreDashboard } from './reports/queryStoreDashboard';
 import { IconPathHelper } from './common/iconHelper';
 
 export async function activate(context: vscode.ExtensionContext): Promise<void> {
-	// TODO: get db name
 	// TODO: add OE entry point with condition for command to only be visible for db's with Query Store enabled (or consider always showing and having a way to enable when dashboard is opened?)
```

Comment:
IMO, always show and have a way to enable when the dashboard is opened.  I would expect to be able to turn Query Store on (and off) in the extension.